specs/commandline/plugin.md

```diff
@@ -12,15 +12,15 @@ Use `notation plugin` to manage plugins. See notation [plugin documentation](htt
 Manage plugins
 
 Usage:
-  notation plugin <command>
+  notation plugin [command]
 
 Available Commands:
   list        List installed plugins
 
 Flags:
   -h, --help          help for plugin
 
-RUN 'notation plugin <command> --help' for more information on a command.
+Use 'notation plugin [command] --help' for more information about a command.
```

Comment:
We can just remove this line as other specs do not have it.

Reply:
Removed this line.